pwa/src/service-worker/set-up-service-worker.js

```diff
@@ -155,6 +161,75 @@ export function setUpServiceWorker() {
         // '[]' fallback prevents an error when switching pwa enabled to disabled
         const sharedBuildManifest = self.__WB_BUILD_MANIFEST || []
         precacheAndRoute(sharedBuildManifest)
+    } else {
+        // This will execute in dev environments
+        precacheAndRoute([
+            // Let this one work
+            { url: './index.html', revision: '1' },
+            // The following are expected to fail
+            { url: 'https://not-a-site.com/hey.jpg', revision: '1' },
+            { url: 'https://not-a-site.com/bogus1.jpg', revision: '1' },
+            { url: 'https://not-a-site.com/derp2.jpg', revision: '1' },
+            { url: 'https://not-a-site.com/diddlibap3.jpg', revision: '1' },
+            { url: 'https://not-a-site.com/squip4.jpg', revision: '1' },
+        ])
+
+        /* An alternative to patch package, where a custom precache controller is made and implemented
```

Comment:
This is included here to show an example of another approach to catching & handling precaching errors (option 2 in the main PR comment)